go/obsclient/authclient_test.go

```diff
@@ -0,0 +1,53 @@
+package obsclient
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/obscuronet/go-obscuro/go/rpcclientlib"
+	"github.com/stretchr/testify/assert"
```

Comment:
Why was `stretchr` chosen in particular? It it the most commonly used?

Reply:
Yeah seems to be the most commonly used, that mock stuff was the most used too.